modules/shortcodes/dailymotion.php

```diff
@@ -327,3 +327,61 @@ function jetpack_dailymotion_embed_reversal( $content ) {
 }
 
 add_filter( 'pre_kses', 'jetpack_dailymotion_embed_reversal' );
+
+/**
+ * DailyMotion Widget Shortcode
+ *
+ * Example:
+ * [dailymotion-widget id=XYZ]
+ *
+ * @param array $atts
+ * @return string html
+ */
+function jetpack_dailymotion_widget_shortcode( $atts ) {
+	if ( isset( $atts['id'] ) ) {
+		$id = $atts['id'];
+	} else {
+		return '<!--Dailymotion error: bad or missing ID-->';
+	}
+
+	$output="<div class='dailymotion-widget' data-placement='" . $id . "'></div>";
```

Comment:
needs spacing around the equal sign. Also, if you're using " you can just write:
```php
$output = "<div class='dailymotion-widget' data-placement='$id'></div>";
```